packages/agent/src/app.ts

```diff
@@ -12,16 +11,27 @@ import {
 } from '@medplum/core';
 import { Endpoint, Reference } from '@medplum/fhirtypes';
 import { Hl7Client } from '@medplum/hl7';
-import { exec as _exec } from 'node:child_process';
+import { ExecException, ExecOptions, exec } from 'node:child_process';
 import { isIPv4, isIPv6 } from 'node:net';
-import { promisify } from 'node:util';
-import { platform } from 'os';
+import { platform } from 'node:os';
 import WebSocket from 'ws';
 import { Channel } from './channel';
 import { AgentDicomChannel } from './dicom';
 import { AgentHl7Channel } from './hl7';
 
-const exec = promisify(_exec);
+async function execAsync(command: string, options: ExecOptions): Promise<{ stdout: string; stderr: string }> {
```

Comment:
Out of curiosity, what does this do differently than `promisify()`?

Reply:
`node` uses a special promisify for a lot of the node core modules, making it hard to mock. The easiest way to mock it would be to mock `exec` directly, but the special internal version doesn't directly wrap `exec` 😬 so it becomes difficult to test without mocking the promisify function too and just is really unclear. This is much simpler than what I tried to do originally with mocking 😅